tests/TestRvsdgs.cpp

```diff
@@ -1288,6 +1288,205 @@ GammaTest::SetupRvsdg()
   return module;
 }
 
+std::unique_ptr<llvm::RvsdgModule>
+GammaTest2::SetupRvsdg()
+{
+  using namespace jlm::llvm;
+
+  auto rvsdgModule = RvsdgModule::Create(util::filepath(""), "", "");
+  auto rvsdg = &rvsdgModule->Rvsdg();
+
+  auto nf = rvsdg->node_normal_form(typeid(rvsdg::operation));
+  nf->set_mutable(false);
+
+  auto SetupLambdaF = [&]()
+  {
+    auto SetupGamma = [](
+      rvsdg::output * predicate,
+      rvsdg::output * xAddress,
+      rvsdg::output * yAddress,
+      rvsdg::output * zAddress,
+      rvsdg::output * memoryState)
+    {
+      auto gammaNode = rvsdg::gamma_node::create(predicate, 2);
+
+      auto gammaInputX = gammaNode->add_entryvar(xAddress);
+      auto gammaInputY = gammaNode->add_entryvar(yAddress);
+      auto gammaInputZ = gammaNode->add_entryvar(zAddress);
+      auto gammaInputMemoryState = gammaNode->add_entryvar(memoryState);
+
+      // gamma subregion 0
+      auto loadXResults = LoadNode::Create(
+        gammaInputX->argument(0),
+        {gammaInputMemoryState->argument(0)},
+        jlm::rvsdg::bit32,
+        4);
+
+      auto one = rvsdg::create_bitconstant(gammaNode->subregion(0), 32, 1);
+      auto storeZRegion0Results = StoreNode::Create(
```

Comment:
Why does the store node not have a memory state?

Reply:
It has a memory state: `{loadXResults[1]},`